utils/mock-data-loader/user-helper.js

```diff
@@ -0,0 +1,36 @@
+const api = require('./api');
+const MOCK_BANKS = require('./banks');
+
+const mockDataLoaderUser = {
+  username: 're-insert-mocks',
+  password: 'AbC!2345',
+  firstname: 'Mock',
+  surname: 'DataLoader',
+  roles: ['editor', 'maker'],
+  email: 're-insert-mocks-data-loader@ukexportfinance.gov.uk',
+  bank: MOCK_BANKS.find((bank) => bank.id === '9'),
+};
+
+const createAndLogInAsInitialUser = async () => {
+  console.info(`try log in as "${mockDataLoaderUser.username}"`);
```

Comment:
We are writing console.info messages without double quotes inside the string, can they all please be removed.